cloudinit/config/cc_disk_setup.py

```diff
@@ -133,8 +133,11 @@ def handle(_name, cfg, cloud, log, _args):
             if not isinstance(definition, dict):
                 log.warning("Invalid disk definition for %s" % disk)
                 continue
-
             try:
+                if definition.get("timeout"):
+                    util.wait_for_files([disk],
```

Comment:
Should this move outside the `try` block here? This call failing would not really be a "Failed partitioning operation", and perhaps should not be ignored in the same way.

Reply:
I took a whack at this, let me know if you think this works better.